config/crd/bases/cloud.spaceship.com_networkattachments.yaml

```diff
@@ -76,65 +80,68 @@ spec:
                   source:
                     type: string
                 required:
+                - bridge
                 - enabled
                 - source
                 type: object
               nodeName:
                 type: string
               nodeSelectors:
                 items:
-                  description: A label selector is a label query over a set of resources.
-                    The result of matchLabels and matchExpressions are ANDed. An empty
-                    label selector matches all objects. A null label selector matches
-                    no objects.
+                  description: |-
+                    A label selector is a label query over a set of resources. The result of matchLabels and
```

Comment:
label selector or node selector?

Reply:
I believe a label selector is used for the node selector, I think this message was generated automatically.

Reply:
so, you select nodes by their labels